Makefile

```diff
@@ -96,4 +96,4 @@ kernel-%: ## Updates the kernel configs: e.g. make kernel-olddefconfig; make ker
 
 .PHONY: conformance
 conformance: ## Performs policy checks against the commit and source code.
-	docker run --rm -it -v $(PWD):/src -w /src ghcr.io/talos-systems/conform:v0.1.0-alpha.22 enforce
+	docker run --rm -it -v $(PWD):/src -w /src ghcr.io/siderolabs/conform:latest enforce
```

Comment:
was changing to the `latest` version really intended?

Reply:
`conform` is for CI tests, shouldn't hurt :sweat_smile:

Reply:
on second thoughts since this is a backport makes sense to only change the org name, thanks for the pointer :+1: